_rules/image-non-empty-accessible-name-23a2a8.md

```diff
@@ -59,6 +59,8 @@ _There are currently no assumptions._
 
 ## Background
 
+Some elements have a role of `button` and a default accessible name defined by the [HTML Accessibility API Mapping][html aam input button], for example `input` elements whose `type` [attribute value][] is either `submit` or `reset`. This rule consider that these default names can be descriptive and therefore does not fail them.
```

Comment:
I think the statement - `This rule consider that the default names can be descriptive and therefore does not fail them.`  I think should be included in the `Assumptions` section as a counter part to the assumption in the `image-button` rule.

Reply:
```suggestion
Some elements have a role of `button` and a default accessible name defined by the [HTML Accessibility API Mapping][html aam input button], for example `input` elements whose `type` [attribute value][] is either `submit` or `reset`. This rule considers that these default names can be descriptive and therefore does not fail them.
```